scipy/stats/tests/test_stats.py

```diff
@@ -1626,6 +1626,21 @@ def test_zscore_ddof(self):
         assert_array_almost_equal(z[1], z1_expected)
 
 
+    def test_mad(self):
+        dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
+                3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
+                3.7, 3.7,3.77, 5.28, 28.95])
+        assert_almost_equal(stats.mad(dat, axis=None), 0.52632)
```

Comment:
I agree with this test. LGTM.  The docstring does not indicate the scale multiplier value of 1.4826 explicitly, would be good to write that number in the docstring somewhere. In trying to verify it took me a bit to figure out the discrepancy of the result with: `np.median(np.abs(dat - np.median(dat)))`

Reply:
Do you mean add the scaling factor to the docstring for the test? I've modified the docstring of the `mad` function to explicitly state the scaling factor.

Reply:
>Do you mean add the scaling factor to the docstring for the test?

Yes, you got it-thanks.